src/components/draw/feature-measurements.vue

```diff
@@ -81,12 +111,21 @@ function onClickValidateRadius() {
       v-if="featureType === 'drawnCircle'"
       class="flex items-center"
     >
-      <span>{{ t('Rayon:') }}</span>
-      <span v-if="!isEditingFeature">{{ featRadius }}</span>
+      <span>{{ t('Rayon:') }} </span>
+      <span v-if="!isEditingFeature" v-format-length="featRadius"></span>
       <!-- Radius is editable when edition mode is on -->
       <div v-else class="flex">
-        <input class="form-control block" type="text" />
-        <button class="lux-btn-primary" @click="onClickValidateRadius">
+        <input
+          data-cy="featItemInputRadius"
+          class="form-control block"
+          type="text"
```

Comment:
Maybe this one should be type number (or maybe not because it is a text in prod). But at least user should not be able to input alpha characters.

![image](https://github.com/user-attachments/assets/c569d9b2-7e3b-4d9a-9b6d-159ff3dee309)

Reply:
Yep, good point, indeed a number field seems much more appropriate here even though production uses text. I've changed it accordingly.